src/templates/static/affiliate_ib_landing.jsx

```diff
@@ -0,0 +1,454 @@
+                                                        <td>{it.L('40 – 59.999%')}</td>
+                                                        <td>{it.L('0.75%')}</td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>{it.L('60 – 79.999%')}</td>
+                                                        <td>{it.L('5%')}</td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>{it.L('80 – 94.999%')}</td>
+                                                        <td>{it.L('0.25%')}</td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>{it.L('95%+')}</td>
+                                                        <td>{it.L('0%')}</td>
+                                                    </tr>
+                                                </table>
+                                            </div>
+                                        </TabContent>
+                                        <TabContent id='cpa'>
+                                            <div className='gr-11 gr-centered text-align-left'>
+                                                <h3 className='no-margin'>{it.L('Cost per acquisition (CPA) for EU affiliates only')}</h3>
+                                                <p>{it.L('Earn USD 100 in commission for each successful referral. Your referred client must open areal money investment account through your unique affiliate link and deposit a total of USD 100 or more (one-time or accumulative) in the account. This commission plan is only available to affiliates based in the EU.')}</p>
+                                                <a href={it.affiliate_signup_url} className='button center-text gr-centered gr-padding-20' target='_blank' rel='noopener noreferrer'>
+                                                    <span>{it.L('Sign up Now')}</span>
+                                                </a>
+                                            </div>
+                                        </TabContent>
+                                    </TabContentContainer>
+                                </div>
+                            </TabContainer>
+                        </div>
+                    </div>
+                    <div id='ib'>
+                        <div className='gr-12 gr-padding-20 white-bg-color'>
+                            <p className='no-margin'>{it.L(`Earn daily commissions on the trading activities of the clients you refer to the ${it.website_name} MetaTrader 5 (MT5) platform. Your total commissions will be credited into your account by the end of the day.`)}</p>
```

Comment:
better to use a placeholder to `it.website_name`

Reply:
pushed